content/en/guides/models/app/features/panels/line-plot/_index.md

```diff
@@ -168,4 +169,52 @@ From a line plot, you can quickly create a [run metrics notification]({{< relref
 1. Hover over the panel, then click the bell icon.
 1. Configure the automation using the basic or advanced configuration controls. For example, apply a run filter to limit the scope of the automation, or configure an absolute threshold.
 
-Learn more about [Automations]({{< relref "/guides/core/automations" >}}).
\ No newline at end of file
+Learn more about [Automations]({{< relref "/guides/core/automations" >}}).
+
+## Visualize CoreWeave infrastructure alerts
+
+{{< alert>}}
+This feature is in Preview and only available when training on a CoreWeave cluster. Contact your W&B representative for access.
+{{< /alert >}}
+
+Observe infrastructure alerts such as GPU failures, thermal violations, and more during machine learning experiments you log to W&B. During a [W&B run]({{< relref "/guides/models/track/runs/_index" >}}), [CoreWeave Mission Control](https://www.coreweave.com/mission-control) monitors your compute infrastructure. If an error occurs, CoreWeave sends that information to W&B. W&B populates infrastructure information onto your run’s plots in your project’s workspace. CoreWeave attempts to automatically resolve some issues, and W&B surfaces that information in the run’s page.
+
+### Find infrastructure issues in a run
+
+W&B surfaces both SLURM job issues and cluster node issues. View infrastructure errors in a run:
+
+1. Navigate to your project on the W&B App. 
+2. Select the **Workspace** tab to view your project’s workspace.
+3. Search and select the name of the run that contains an infrastructure issue. One or more red vertical lines with an exclamation mark overlay plots indicating an infrastructure issue. 
+4. Select an issue on a plot or select the **Issues** button in the top right of the page. A modal appears that lists each issue reported by CoreWeave. 
+
+{{< alert title="Tip" >}}
+Pin the **Issues** column to your W&B Workspace to view runs that logged an issue at a glance. For more information about how to pin a column, see [Customize how runs are displayed]({{< relref "/guides/models/track/runs/#customize-how-runs-are-displayed" >}}).
+{{< /alert >}}
+
+The **Overall Grafana view** at the top of the modal redirects you to the SLURM job's Grafana dashboard, which contains system-level details about the run. The **Issue** summary describes the root error that the SLURM job reported to CoreWeave Mission Control. The summary section also describes any attempts to automatically resolve the error made by CoreWeave.
```

Comment:
If the section is labeled Summary, should it be bold?

For a followup, do you think an image of the dashboard would be helpful?